components/ServicesBlurbs.vue

```diff
@@ -1,153 +1,110 @@
-              user friendly. Stay ahead of tech giants and build a solid
-              foundation for your digital future with Ship Shape’s Nuxt
-              consulting team.
-            </p>
-
-            <nuxt-link class="learn-more" to="/services/nuxt-consulting/">
-              Learn more
-              <inline-svg class="h-4 inline w-4" src="/svgs/angle-right.svg" />
-            </nuxt-link>
-          </div>
-        </div>
-
-        <div
-          class="bg-white flex flex-grow p-8 justify-center xl:h-104 lg:p-12"
-          data-aos="fade-up"
-          itemscope
-          itemtype="http://schema.org/Service"
-        >
-          <div class="flex flex-col max-w-2xl w-full">
-            <h2 class="mb-6 mt-4 text-3xl" itemprop="name">App Tours</h2>
-
-            <p class="flex-grow">
-              We built our own open source library,
-              <a href="https://shepherdjs.dev/">Shepherd.js</a> to guide users
-              through the features and functions of your web app. It’s no
-              surprise that we’ve been tapped to help companies of all sizes to
-              implement user-friendly app tours.
-            </p>
-
-            <nuxt-link class="learn-more" to="/services/app-tours/">
-              Learn more
-              <inline-svg class="h-4 inline w-4" src="/svgs/angle-right.svg" />
-            </nuxt-link>
-          </div>
-        </div>
-
-        <div
-          class="bg-white flex flex-grow p-8 justify-center xl:h-104 lg:p-12"
-          data-aos="fade-up"
-          itemscope
-          itemtype="http://schema.org/Service"
-        >
-          <div class="flex flex-col max-w-2xl w-full">
-            <h2 class="mb-6 mt-4 text-3xl" itemprop="name">
-              Mentoring and Team Training
-            </h2>
-
-            <p class="flex-grow">
-              Whether you're a Senior Developer or Junior Developer, Ship Shape
-              fosters one-on-one connections that help make a difference in our
-              community. We offer several Ember mentorship programs that
-              identify your goals and prepare you and your team for the future
-              of Ember.
-            </p>
-
-            <nuxt-link class="learn-more" to="/services/mentoring/">
-              Learn more
-              <inline-svg class="h-4 inline w-4" src="/svgs/angle-right.svg" />
-            </nuxt-link>
-          </div>
-        </div>
-      </div>
+          <template #header>
+            <inline-svg :class="service.logoClasses" :src="service.logoSrc" />
+          </template>
+          <template #description>
+            {{ service.description }}
+          </template>
+        </ServiceCard>
+      </section>
     </div>
-  </div>
+  </section>
 </template>
+
+<script>
+const headerImage = { src: '/img/hero-images/specialties.png', height: '814' };
+
+const services = [
+  {
+    logoClasses: 'h-auto w-20 nested-svg pb-6 lg:w-32 lg:pb-12',
```

Comment:
It looks like all of these have at least `h-auto` and `nested-svg`. Could we move those to be directly on the element?

Reply:
I thought about having some base classes on the element and the dynamic classes (padding/width) separate but then it'd be both `class` and `:class` for each so I thought this would be simpler. Let me know if you still would like it changed.

Reply:
Having both `class` and `:class` is definitely valid. I've done that in many places. If you don't mind, I think that would be a little cleaner to put the common classes in `class` and use `:class` for dynamic ones.